packages/node/src/breadcrumbs/FileBreadcrumbsStorage.ts

```diff
@@ -68,6 +62,19 @@ export class FileBreadcrumbsStorage implements BreadcrumbsStorage {
         ];
     }
 
+    public getAttachmentProviders(): BacktraceAttachmentProvider[] {
+        return [
+            {
+                get: () => new BacktraceFileAttachment(this._mainFile, 'bt-breadcrumbs-0'),
```

Comment:
I'm not fully sure what it means that the attachment is dynamic in this context. I was sure the attachment is dynamic when the attachment context is in the app memory. breadcrumbs are not in the memory. Why it's dynamic?

Reply:
The files may not exist at a given time. Also, with the changes made in breadcrumb limiting, this will not work, as file names are incremented instead of switched.